src/Map/src/Polygon.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\Map;
+
+/**
+ * Represents a polygon on a map.
+ *
+ * @author [Pierre S]
+ */
+final readonly class Polygon
+{
+    /**
+     * @param array<string, mixed> $extra Extra data, can be used by the developer to store additional information and use them later JavaScript side
+     */
+    public function __construct(
+        private array $points,
+        private ?string $title = null,
+        private ?array $rawOptions = null,
```

Comment:
`rawOptions` was supposed to be a _JavaScript-only_ parameter, but I believe it can makes sense to also have it here... 
Can you remove `rawOptions` from this PR, so we can add it back (to Marker/InfoWindow/Polygon) in another PR if devs have this use-case? 🙏🏻

Reply:
Sure i can, but it was very helpful, example to change styles for polygons inside the controller

Reply:
Thanks!